serverlessworkflow/antora.yml

```diff
@@ -11,13 +11,14 @@ asciidoc:
     quarkus_cli_platform_redhat: ""
     # upstream: remove this var
     #kogito_version_redhat: 1.24.0.Final-redhat-00001
+    kogito_version_redhat: ~
     # upstream: io.quarkus.platform
     #quarkus_platform: com.redhat.quarkus.platform
     quarkus_platform: io.quarkus.platform
     # upstream: kogito-quarkus-serverless-workflow
     # kogito_sw_gav: org.kie.kogito:kogito-quarkus-serverless-workflow:1.24.0.Final-redhat-00001
     kogito_sw_gav: kogito-quarkus-serverless-workflow
-    
+    # downstream: 2.7.6.Final-redhat-00006
```

Comment:
what is `downstream` ?

Reply:
OpenShift Knative docs. :P
Should I change it?